### DISCUSSION TITLE
【读书会】《解读基金》 - 第1期 | ApacheCN

### DISCUSSION THREAD

**Discussion by yanqiang921106** [apachecn/home#233] [Ideas]
# <center>【读书会】《解读基金》 - 第1期 | ApacheCN<center>

* 【推荐】读书会: 《解读基金》季凯帆
* 【推荐】留言地址: https://github.com/apachecn/home/issues/194

> 参与

* QQ群【临时群】: 1062944139
* 负责人紫色木槿: 1076003661 (QQ)

> 时间

**每两周三晚8点，月初会发布本月阅读书籍2-4本，顺序阅读。

* 时间周期: 2021-06-09 ～ 2021-06-23 （共 14 天，对自己严格的建议控制在7天内）
* 语音频率: 每两周三晚9点语音一次，周末休息跳过
* **群友发言：**5~10人，5分钟/人，计划时长0.5-1小时，带着问题发言	
* **[@紫色木槿](https://github.com/yanqiang921106)** 会进行语音: 沟通进度和督促学习

> 内容

1. 分享读书心得
2. 学到的知识
3. 可以应用到实际工作生活中的
4. 自己的应用计划			

每一本书，专门一个issues，供大家留言分享		

> 活动目的

1. 培养自己对理财的正确认识
2. 选出2个人作为大肥羊学校的学员，作为其他活动的代理班长		

---

* 总结：一人可以走的很快，但是一群人却可以走的很远

**Comment by yanqiang921106** [comment]
问题：拿多少钱投资？
答：1、三分之一用来买房地产，三分之一投资证券市场，三分之一用来储蓄或者买国债；
2、指头算法：可用于各类投资的资产比例=100-你目前的年龄，年龄越大越保守，年纪越轻越可以进取

**Comment by yanqiang921106** [comment]
问：复利计算的“72法则”、“115法则”是什么？
答：假设投资收益率为X，X为12%，则投资翻倍需要的时间约为72/12=6年，投资变为3倍的时间为115/12=9.6年

**Comment by yanqiang921106** [comment]
问：如何选择基金品种？
答：1、首先选择优秀的基金公司，基金经理。可以根据公司下所有基金的晨星评级均值，选择较高的。基金经理选择有长期从业经验，收益较高，且跳槽没那么频繁的。
2、基金的资产配置，基金招募书中有，股票占比越大越激进
3、技术指标：
--收益
 ①平均回报，代表收益，越大越好；
 ④阿尔法系数，代表基金能在多大程度上跑赢整个市场，越大越好
--风险
 ②标准差，代表基金波动情况
 ⑤贝塔系数，表现基金相对于大盘的波动情况，指数基金为1，大于1代表比大盘波动大
--收益风险综合
 ③夏普比率，综合收益和风险系数；夏普比率高，在收益相同的情况下，波动低；反之，在风险相同的情况下收益就高
--其他
 ⑥R平方，代表基金和大盘的相关性。如果是1，表示和大盘完全相关；如果R平方和1相差很大，贝塔系数就没有什么意义了

注意：必须要在同类基金中进行比较，也就是相对于同样的资产组合标准！！！！
基金评级网站：晨星 https://www.morningstar.cn/main/default.aspx

**Comment by yanqiang921106** [comment]
问：基金规模越大越好么
答：不是，规模是业绩的敌人。
![image](https://user-images.githubusercontent.com/34933285/120879016-f5b57700-c5f2-11eb-911e-abe86bd971d1.png)

**Comment by yanqiang921106** [comment]
问：什么时候赎回基金？
答：①这只基金不再符合自己的投资策略；
②自己的投资目标已经达到。
注意：关注基金的份额大规模减少。因为太小的基金将面临清盘关闭的风险。

**Comment by yanqiang921106** [comment]
问：投资组合是根据投资策略制定的，投资策略需要考虑哪几个方面？
答：①投资期
②投资者的心理风险承受能力
③投资者的心理投资回报预期

**Comment by yanqiang921106** [comment]
问：选择一次性投资还是定投？
答：①一次性投资：~。~
注意：假设有一笔100万资金入账，大可不必分10年定投，可以分两三次买入或一次性买入
②定投：定投的标准说法是定时定额投资。也就是说，咱们在一个固定的日子，对某一基金投入固定的资金
定投的优点：①长期定投，平滑市场波动 ②适合每个月结余的同学 ③无须考虑市场情况

**Comment by yanqiang921106** [comment]
收费方式的选择：长期--后端收费，短期--前端收费
分红方式的选择：分红再投资

**Comment by yanqiang921106** [comment]
买入策略：
1、定投。按月按周定投，区别不是很大。可以用历史数据进行回测
回测地址：http://data.eastmoney.com/money/calc/FundCalcDTSY.html
2、考虑投资目标：
①养老：30年以上投资期
投资组合可以相对激进，100%股票基金
②子女教育：15~20年投资期
80%股票基金+20%债券基金
③5年投资期：5年以下投资期
80%债券基金+20%股票基金

**Comment by yanqiang921106** [comment]
卖出策略：
（1）如果我的投资目标变了，我肯定要改变投资组合
（2）如果基金公司发生变化
（3）如果发现犯了错误，会立刻改正错误：例如基金规模不到1亿
（4）在购买的时候，我选择的基金都是四或五星的，但基金连续3~6个月以上评级跌到3星的
（5）我不会因为基金的绝对涨跌幅度来决定买卖，也不设置一个止损线。
（6）如果一只基金出现问题，我会去选择一个替代品，把它转换过去。
（7）市场表现太疯狂了，也要赎回基金。如估值、估值百分比和“且慢指数估值”，貌似很容易判断市场在高位还是在低位
估值地址：https://qieman.com/idx-eval
理念：不预测市场，主要看基金公司的基本面。

**Comment by jiangzhonglian** [comment]
1
